modules/edw_event/modules/edw_event_groups/edw_event_groups.routing.yml

```diff
@@ -14,6 +14,23 @@ edw_event_groups.meeting_access_groups.settings:
           - event
     _admin_route: TRUE
 
+edw_event_groups.meeting_contributors.listing:
+  path: '/node/{node}/meeting_contributors'
```

Comment:
Please replace with `'/node/{node}/meeting-contributors'`. Same for `edw_event_groups.remove_group_from_section` route. And in any other places (e.g. view)